### ISSUE TITLE
Support osxfuse 3

### ISSUE THREAD

**Issue by DrLulz** [GoogleCloudPlatform/gcsfuse#195]
`osxfuse 3.5.2`
`gcsfuse 0.19.0`

My system contains a `/Library/Filesystems/osxfuse.fs/Contents/Resources/load_osxfuse`, but not a `/Library/Filesystems/osxfusefs.fs/Support/load_osxfusefs`. Please advise.

```
$ gcsfuse photos-archive /Users/DrLulz/Desktop/mnt/gcs-bucket/
Using mount point: /Users/DrLulz/Desktop/mnt/gcs-bucket
Opening GCS connection...
Opening bucket...
Mounting file system...
daemonize.Run: readFromProcess: sub-process: mountWithArgs: mountWithConn: Mount: mount: loadOSXFUSE: fork/exec /Library/Filesystems/osxfusefs.fs/Support/load_osxfusefs: no such file or directory
```

**Comment by jacobsa** [comment]
jacobsa/fuse#18 should fix this. Will need to import the package after that is committed.

**Comment by jacobsa** [comment]
Done in the above commit. Now need to perform a release.

**Comment by jacobsa** [comment]
Should be fixed in v0.20.0. I've opened homebrew/homebrew-fuse#77 for getting it in homebrew.
